src/palace/manager/feed/serializer/opds.py

```diff
@@ -405,13 +413,95 @@ def _serialize_data_entry(self, entry: DataEntry) -> etree._Element:
         return cast(str, etree.tostring(element, encoding="unicode"))
 
+    @abc.abstractmethod
     def content_type(self) -> str:
-        return OPDSFeed.ACQUISITION_FEED_TYPE
+        pass
+
+    @abc.abstractmethod
+    def _serialize_facet_links(self, feed: FeedData) -> list[Link]:
+        pass
+
+    @abc.abstractmethod
+    def _serialize_sort_links(self, feed: FeedData) -> list[Link]:
+        pass
+
+
+class OPDS1Version1Serializer(BaseOPDS1Serializer):
+    """An OPDS 1.2 Atom feed serializer.  This version of the feed implements sort links as
+    facets rather than using the http://palaceproject.io/terms/rel/sort rel and does not  support
+    the http://palaceproject.io/terms/properties/default property indicating default facets
+    """
+
+    @override
+    def _serialize_facet_links(self, feed: FeedData) -> list[Link]:
+        links = []
+        if feed.facet_links:
+            for link in feed.facet_links:
+                links.append(self._serialize_feed_entry("link", link))
+        return links
+
+    @override
+    def _serialize_sort_links(self, feed: FeedData) -> list[Link]:
+        # Since this version of the serializer implements sort links as facets,
+        # we return an empty list of sort links.
+        return []
+
+    @override
+    def _get_attribute_mapping(self) -> dict[str, str]:
+        return V1_ATTRIBUTE_MAPPING
+
+    @override
+    def content_type(self) -> str:
+        return OPDSFeed.ACQUISITION_FEED_TYPE + "; api-version=1"
+
+
+class OPDS1Version2Serializer(BaseOPDS1Serializer):
+    """An OPDS 1.2 Atom feed serializer with Palace specific modifications (version 2) to support
+    new IOS and Android client features. Namely, this version of the feed implements sort links as
+    links using the http://palaceproject.io/terms/rel/sort rel.  The active or selected sort link is indicated
+    by the http://palaceproject.io/terms/properties/active-sort property.  Default facets and sort links are
+    inidcated by the http://palaceproject.io/terms/properties/default property.
+    """
+
+    @override
+    def _serialize_facet_links(self, feed: FeedData) -> list[Link]:
+        links: list[Link] = []
+        facet_links = feed.facet_links
+        if facet_links:
+            for link in facet_links:
+                # serialize all but the sort facets.
+                if not is_sort_facet(link):
+                    links.append(self._serialize_feed_entry("link", link))
+        return links
+
+    @override
+    def _serialize_sort_links(self, feed: FeedData) -> list[Link]:
+        links: list[Link] = []
+        facet_links = feed.facet_links
+        if facet_links:
+            for link in feed.facet_links:
+                # select only the sort facets for serialization
+                if is_sort_facet(link):
+                    links.append(self._serialize_sort_link(link))
+        return links
 
     def _serialize_sort_link(self, link: Link) -> etree._Element:
```

Comment:
We aren't using the `@override` decorator anywhere else in the codebase currently. I think if we're going to apply it in these classes we should apply it consistently, so it should get added to this method as well.